src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -508,13 +552,68 @@ describe('NewWorkspaceModal', () => {
         );
       });
 
-      await user.click(screen.getByText('Select a billing project'));
-      await user.click(screen.getByText('Azure Billing Project'));
+      await selectBillingProject(user, 'Azure Billing Project');
 
       // Assert
-      expect(screen.queryByText('Policies')).toBeNull();
+      expect(screen.queryByText(policyTitle)).toBeNull();
       expect(screen.queryByText(policyLabel)).toBeNull();
     });
+
+    it('Allows toggling PHI tracking from an enterprise protected data billing project if cloned workspace does not have PHI tracking', async () => {
+      // Arrange
+      const user = userEvent.setup();
+      setup({ billingProjects: [azureProtectedEnterpriseBillingProject] });
+
+      // Act
+      await act(async () => {
+        render(
+          h(NewWorkspaceModal, {
+            cloneWorkspace: defaultAzureWorkspace,
+            onDismiss: () => {},
+            onSuccess: () => {},
+          })
+        );
+      });
+
+      await selectBillingProject(user, 'Enterprise Azure Billing Project');
+
+      // Assert
+      screen.getByText(policyTitle);
+      // Make sure we don't show both the read-only checkbox and the one that can be toggled.
+      const checkboxes = screen.getAllByRole('checkbox', { name: phiTrackingLabel });
+      expect(checkboxes.length).toBe(1);
+      const checkbox = checkboxes[0];
+      expect(checkbox).not.toHaveAttribute('disabled');
```

Comment:
Will the `toBeDisabled` matcher work here?
https://github.com/testing-library/jest-dom?tab=readme-ov-file#tobedisabled

```suggestion
      expect(checkbox).not.toBeDisabled();
```

Reply:
Hmmmm, I don't think so, but I will try it.

Reply:
@nawatts It is not good to use because it doesn't actually look for the way that we disable checkboxes. That is, on a disabled checkbox `toBeDisabled` does not return true, as shown in the example below.

```
    Received element is not disabled:
      <span aria-checked="true" aria-describedby="element-365" aria-labelledby="element-366" class="terra-ui--interactive fa-layers fa-fw" disabled="" role="checkbox" style="vertical-align: middle; margin: 0rem 0.25rem 0.25rem 0rem; fill: var(--hover-color, undefined); cursor: not-allowed;" tabindex="-1" />

Reply:
Argh. I wonder if this is actually exposing an issue with our checkbox component. But that's a task for another day.